app/src/main/java/com/klinker/android/twitter/activities/compose/Compose.java

```diff
@@ -763,16 +772,17 @@ private void startUcrop(Uri sourceUri) {
     public static final int PWICCER = 420;
 
     public boolean pwiccer = false;
+    private boolean multiTweets = true;
```

Comment:
Fixed. Good to have your eyes there. Sometimes one just concentrates on the task at hand ;). The next PR will hopefully have a setting in UI where the feature can be enabled or disabled.